_staff_members/02_fausto.md

```diff
@@ -4,5 +4,6 @@ position: Master of the frontend
 image_path: images/team/fausto_nunez_bw_sq_sm.jpg
 github: brainlessdeveloper
 vim_config: https://github.com/brainlessdeveloper/vimrc/blob/master/init.vim
+personal_website: http://brainlessdeveloper.com
```

Comment:
`https` tho

Reply:
Ohh, didn't know It had `https` - nice 💯- updating that right away